app/javascript/pages/dashboards/header/header-actions.js

```diff
@@ -33,27 +33,44 @@ export const getHeaderData = createThunkAction(
           ) => {
             const extent = totalExtent.data.data;
             const loss = totalLoss.data.data;
-            const plantationsExtent = totalPlantationsExtent.data.data;
             const plantationsLoss = totalPlantationsLoss.data.data;
-            const primaryLoss = totalPrimaryLoss.data.data;
+            const plantationsExtent = totalPlantationsExtent.data.data;
+
+            // group over years
             const groupedLoss = loss && groupBy(loss, 'year');
+            const groupedPlantationsLoss =
+              plantationsLoss && groupBy(plantationsLoss, 'year');
+            const groupedPlantationsExtent =
+              plantationsExtent && groupBy(plantationsExtent, 'year');
+
+            const primaryLoss = totalPrimaryLoss.data.data;
             const latestYear = max(Object.keys(groupedLoss));
+
+            // sum over different bound1 within year
+            const summedPlantationsLoss =
+              groupedPlantationsLoss &&
+              sumBy(groupedPlantationsLoss[latestYear], 'area');
+            const summedPlantationsEmissions =
+              groupedPlantationsLoss &&
+              sumBy(groupedPlantationsLoss[latestYear], 'emissions');
+            const summedPlantationsExtent =
+              groupedPlantationsExtent &&
+              sumBy(groupedPlantationsExtent[latestYear], 'area');
             const summedLoss = sumBy(groupedLoss[latestYear], 'area');
             const summedEmissions = sumBy(groupedLoss[latestYear], 'emissions');
             const data = {
               totalArea: (extent[0] && extent[0].total_area) || 0,
```

Comment:
Again here we aren't checking if extent is an array or even if it exists. Can we improve this?

Reply:
This still needs addressing. What is the extent isnt an array? This will crash.

Reply:
plantationsExtent is always a length = 1 array so i've gone with:

`plantationsExtent && plantationsExtent.length ? plantationsExtent[0].area : 0,`